### PR TITLE
Fix deprecated and move to unified_receipt

### PR DESCRIPTION
**Pull request by Mihoid**
Deprecated: latest_receipt, latest_receipt_info, latest_expired_receipt, latest_expired_receipt_info.

According [Apple documentation](https://developer.apple.com/documentation/appstoreservernotifications/responsebody) _latest_receipt, latest_receipt_info, latest_expired_receipt, latest_expired_receipt_info_ are unavailable (since 10 March 2021) in JSON response. 

This PR will fix that.

### CODE REVIEW

**Comment by gabrielgarza** [comment]
@Mihoid Ni catch, let me review real quick

File: lib/monza/status_response.rb [hunk 1]
```diff
@@ -116,10 +128,14 @@ def initialize(attributes)
         @expires_date = DateTime.parse(latest_receipt_info['expires_date_formatted'])
       end
       if latest_receipt_info['expires_date']
-        @expires_date_ms = Time.zone.at(latest_receipt_info['expires_date'].to_i / 1000)
+        @expires_date_ms = if latest_receipt_info['expires_date'].size == 13
+                             Time.zone.at(latest_receipt_info['expires_date'].to_i / 1000)
+                           else
+                             DateTime.parse(latest_receipt_info['expires_date'])
```
Comment (gabrielgarza) [inline lib/monza/status_response.rb:134]: @Mihoid did you find an edge case that needed this?

**Comment by gabrielgarza** [comment]
@Mihoid There was a big bug here that I missed in my review. These deprecations ony apply to Server Event response payloads. 

The response body of the normal verifications has not changed: https://developer.apple.com/documentation/appstorereceipts/responsebody

I've pushed a fix in version 1.5.0 and yanked the buggy 1.4.0 version from Rubygems. 

Please use version 1.5.0

**Comment by Mihoid** [comment]
@gabrielgarza Thanks for fix! It was my mistake.

**Pull request merged**